.release/buildspec_e2e.yml

```diff
@@ -31,6 +33,12 @@ batch:
           TEST_SUITE: customized-env
           APP_REGION: us-west-1
           APP_ACCOUNT: e2e/account/customized_env
+          TESTENV_REGION: us-west-1
```

Comment:
Does it make more sense to combine `TEST_REGION` and `TESTENV_REGION`?

Reply:
good catch, those were typos. `TEST_REGION` was supposed to be  `TESTENV_REGION`